lang/en/content.php

```diff
@@ -174,4 +174,53 @@
             "title_1" => "Are we looking for you?",
         ],
     ],
+    "contact" => [
+        "title_1" => "Write",
+        "title_2" => "to Us!",
+        "company" => "Blumilk Sp. z o.o.",
+        "subtitle" => "Are you interested enough to want to discuss your project with us, exchange ideas, become our business partner, or work with us? We would love to talk with you!",
+        "form" => [
+            "email" => "E-mail",
+            "title" => "Topic",
+            "message" => "Message",
+        ],
+        "location" => [
+            "address" => "ul. Najświętszej Marii Panny 5f/5",
+            "PostalCode" => "59-220 Legnica",
+        ],
+        "phone" => "+48 780 142 367",
+        "email" => "office@blumilk.pl",
+        "info" => [
+            "NIP" => "NIP 6912551135",
+            "KRS" => "KRS 0000866755",
+            "REGON" => "REGON 38737620",
```

Comment:
I would like to keep only these numbers:
```suggestion
            "NIP" => "6912551135",
            "KRS" => "0000866755",
            "REGON" => "38737620",
```

Reply:
This is change merged from #259. Can You request the change there? Also in this case would you like to split it in `"REGON" => [
"description" => "REGON,
"number" =>  "321312413"]`
Because we are displaying the information besides eachother.